inst/@sym/horzcat.m

```diff
@@ -62,7 +62,7 @@
           'return sp.Matrix.hstack(*_proc),'
           };
 
-  varargin = sym(varargin);
+  varargin = cell_array_to_sym(varargin);
```

Comment:
mm, seems addpath its a bad idea, it expose the private functions in the user env, in we use it we will need use addpath in the start in all methods and in the end use rmpath.